tiflash/monitor-tiflash.md

```diff
@@ -47,6 +47,15 @@ The following sections introduce the default monitoring information of **TiFlash
 - Cop task memory usage: The total memory usage of all TiFlash instances processing coprocessor requests.
 - Handling Request Number: The total number of all TiFlash instances processing coprocessor requests. The classification of the requests is the same as that of Request QPS.
 
+## Task Scheduler
+
+- Min TSO: The minimum TSO of all running queries on each TiFlash instance. TiFlash maintains this value to make sure that every query with smaller TSO will be scheduled to run. If there is no running queries at a given time, this value is the max value of type `uint64`.
+- Estimated Thread Usage and Limit: The estimated thread count used by all running queries, and the soft and hard limit of thread count, on each TiFlash instance.
+- Active and Waiting Queries Count: The running and waiting query count on each TiFlash instance.
+- Active and Waiting Tasks Count: The running and waiting task count on each TiFlash instance.
+- Hard Limit Exceeded Count: How many times the estimated thread usage by all running queries has exceeded the hard limit of thread count, on each TiFlash instantce.
+- Task Waiting Duration: The duration of each task waiting to be scheduled after its initialization, on each TiFlash instance.
```

Comment:
```suggestion
- Min TSO: The minimum TSO among all queries running on each TiFlash instance. This configuration ensures that queries with the minimum TSO indicated by this configuration can be scheduled. When no queries are running, this value is the max value of type `uint64`.
- Estimated Thread Usage and Limit: The estimated amount of threads used by all queries running on each TiFlash instance, and the soft and hard limits on the amount.
- Active and Waiting Queries Count: The amount of running queries and that of waiting queries on each TiFlash instance.
- Active and Waiting Tasks Count: The amount of running tasks and that of waiting tasks on each TiFlash instance.
- Hard Limit Exceeded Count: Times that the estimated amount of threads used by queries running on each TiFlash instance exceeds the hard limit.
- Task Waiting Duration: The duration from task initialization to task scheduling on each TiFlash instance.
```
1. this value is the max value of type `uint64`----Does the max value equal to `unit64`?
2. soft and hard limit of thread count---- soft limit and hard limit (two limits), or soft and hard limit (one limit)?
3. each TiFlash instance or a TiFlash instance?

Reply:
0. `This configuration ensures that queries with the minimum TSO indicated by this configuration can be scheduled`: It is not a "configuration", it's an internal state we keep watching and updating all the time (that's why it's a metric). And why we watch and update it? Because we want to `make sure that every query with smaller TSO will be scheduled to run`. 
1. `uint64` is not a value, it's a data type of "unsigned 64-bit integer". Maybe I should have call it that.
2. Two limits. It should be plural, good catch.
3. Every metric is for "each TiFlash instance".